manifest.json

```
@@ -0,0 +1,9 @@
+{
+  "icons": [
+  {
+    "src": "\/android-icon-192x192.png",
```

```suggestion
    "src": "img/favicons/android-icon-192x192.png",
```

А также сам файл должен иметь название **manifest.webmanifest**, а не **manifest.json**.